code/game/objects/items.dm

```diff
@@ -196,7 +197,7 @@
 		return
 
 	pickup(user)
-	if(!user.put_in_active_hand(src))
+	if(!user.put_in_active_hand(src) || !remove_successful)
```

Comment:
why?

Reply:
Unless I want to refactor item code completely this is the only reliable way to ensure the item gets dropped

Reply:
> 
> 
> Unless I want to refactor item code completely this is the only reliable way to ensure the item gets dropped

i think this is good, some storage stuff tends to bug often, i'll see if too much necessary but at all seems like good idea